tests/client.py

```diff
@@ -99,7 +99,11 @@ def send(self, request, *args, **kwargs):
         response_started = False
         raw_kwargs = {"body": io.BytesIO()}
 
-        loop = asyncio.get_event_loop()
+        if not asyncio.get_event_loop().is_closed():
+            loop = asyncio.get_event_loop()
+        else:
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
```

Comment:
Is this change neccessary?

Reply:
yes if we keep the  `test_run_signal_multi_threads`.

if this isn't modified here's what happens:
================================================================ 80 failed, 92 passed, 12 warnings in 9.79s ================================================================

this happens on all tests after this one, because the `loop = asyncio.get_event_loop(`) I replaced above is closed now so you get a `E   RuntimeError: Event loop is closed` for the 80 failed tests after 

should we skip this test then this is not required but my personal guess is that it's a good example